.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -267,6 +303,17 @@ dynamodb_Usage_BatchGetWriteDelete:
   synopsis: get, write, and delete batches of &DDB; items.
   category: Scenarios
   languages:
+    .NET:
+      versions:
+        - sdk_version: 3
+          github: dotnetv3/dynamodb/scenarios
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - DynamoDB.dotnetv3.dynamodb-basics.BatchWriteItem
+                - DynamoDB.dotnetv3.dynamodb-basics.QueryItems
+                - DynamoDB.dotnetv3.dynamodb-basics.ScanTable

```

Comment:
These excerpts should be part of the getting started scenario, not separated into this other scenario.